src/style.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+  box-sizing: border-box;
+}
+
+form {
+  width: 100%;
+}
+
+.container {
+  width: 100%;
+  margin: 20px 0;
+
+}
+
+.searchbar {
+  width: 100%;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  border: 1px solid #e1e7ed;
```

Comment:
It's netter to set default font-style to html tag and use font-family: inherit for inputs
Also you should add correct font-weight value for inputs (for setting correct font styles)